src/types/onyx/PolicyEmployee.ts

```diff
@@ -14,6 +14,9 @@ type PolicyEmployee = OnyxCommon.OnyxValueWithOfflineFeedback<{
     /** Email of the user this user submits all reports to */
     submitsTo?: string;
 
+    /** Email of the user this user forwards all reports when the amount of the report is over limited */
```

Comment:
Sorry but 2 comments here:
1. The `forwardsTo` comment is now not 100% correct :D Maybe we can change it to:
    - Email of the user this user forwards all approved reports to (when report total under 'approvalLimit' or when 'overLimitForwardsTo' is not set)
2. Can we also add a possible key 'approvalLimit' (optional) with some help text like:
    - Determines if this employee should approve a report. If report total > approvalLimit, next approver will be 'overLimitForwardsTo', otherwise 'forwardsTo'
3. Finally, let's update the help text for 'overLimitForwardsTo' to something like:
    - Email of the user this user forwards all reports to when the report total is over the 'approvalLimit'

Reply:
@Beamanator I updated.